test/integration/pj-rehearse/expected.yaml

```diff
@@ -1760,7 +1760,7 @@
       prow.k8s.io/refs.pull: "1234"
       prow.k8s.io/refs.repo: release
       prow.k8s.io/type: presubmit
-    name: test-prowjob
+    name: 52b7ac34-ce00-11ea-b9f1-3e22fb21a9dc
```

Comment:
These fixture changes are wrong

Reply:
That happened when i ran the make update-integration.. I'll try running it again

Reply:
i got rid of the update integration run changes.. and the integration seems to pass without that